### DISCUSSION TITLE
Array of types in migrations

### DISCUSSION THREAD

**Discussion by tavaresgerson** [adonisjs/core#3725] [Help] [answered]
Hello guys!

Its possible create an array of types, for example: `string[]`, `integer[]` for use in Postgres database.

Thanks for your help!

**Comment by tavaresgerson** [comment, marked as answer]
So, it's possible with `table.specificType`.

For example, an array of type double:
```ts
table.specificType('first_coord', 'double precision[]')
```

hope this helps someone 🎧

**Reply by florian-lefebvre** [reply]
I had to add the following properties to the column decorator:
```ts
  @column({ 
     prepare: (value: string[]): string => `{${value.join(',')}}`,
     consume: (value: string): string[] => value.slice(1, -1).split(','), 
   })
```

**Reply by tavaresgerson** [reply]
Yeah, this is an alternative. Thanks bro
